log-manager/src/main/java/io/airlift/log/Logging.java

```diff
@@ -142,6 +142,7 @@ private static void redirectStdStreams()
 
     private synchronized void enableConsole()
     {
+        log.info("Enabling stderr output");
```

Comment:
I would not add this log message.  It also happens before the appender is  added, so it would not go anwhere.

@electrum I think we should inline this method. It is private and only called in the constructor.

Reply:
removed it